concourse/templates/default.mako

```diff
@@ -385,7 +385,7 @@ else:
 % elif job_step.name == 'update_component_dependencies':
         ${update_component_deps_step(job_step=job_step, job_variant=job_variant, github_cfg_name=github.name(), indent=8)}
 % elif job_step.name == 'publish':
-    ${publish_step(job_step=job_step, job_variant=job_variant)}
+        ${publish_step(job_step=job_step, job_variant=job_variant, indent=0)}
```

Comment:
why did you change the indentation here?

Reply:
Probably for consistency? see line 386

Reply:
good point